Shared/messages/MessageFeed.h

```diff
@@ -64,18 +65,20 @@ class MessageFeed : public Esri::ArcGISRuntime::DynamicEntityDataSource
   void setThumbnailUrl(const QUrl& thumbnailUrl);
 
   bool addMessage(const Message& message);
+  Esri::ArcGISRuntime::DynamicEntity* getDynamicEntityById(quint64 entityId);
```

Comment:
Can this also be const? If you're doing any caching stuff, consider making `m_dynamicEntities` mutable so the getter methods can be const.

Reply:
I've updated a few methods use here const where possible. but returning a const DynamicEntity* doesn't work on the getter methods ultimately because the constructor for GeoElementAlertTarget which was the primary reason for exposing the entities this way, needs non-const pointer. Maybe that constructor should be changed?